deploy/TIP.html

```diff
@@ -62,9 +62,9 @@
         <title>Time In Process (TIP) Chart</title>
         
         <!-- HighCharts -->
-        <script type="text/javascript" src="https://people.rallydev.com/js/jquery.min.js"></script>
-        <script type="text/javascript" src="https://people.rallydev.com/js/highcharts.js"></script>
-        <script type="text/javascript" src="https://people.rallydev.com/js/exporting.js"></script>
+        <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.6.1/jquery.min.js"></script>
+        <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.6.1/highcharts.js"></script>
```

Comment:
404's. 
Is CA fine with using the HighCharts CDN directly? 
http://code.highcharts.com/2.1.6/highcharts.js